public/subscribe.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
```

Comment:
I'm guessing this isn't actually working right now. If it is, let's talk about how sending email works from your server